BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -358,12 +358,12 @@ bool is_valid_face_descriptor( typename boost::graph_traits<FaceGraph>::face_des
 
 /*!
   \ingroup PkgBGLHelperFct
- * \brief is_valid_face_graph checks the integrity of a `HalfedgeGraph`.
- * \param g the `HalfedgeGraph` to test.
+ * \brief checks the integrity of `g`.
+ * \param g the HalfedgeGraph to test.
```

Comment:
`HalfedgeListGraph` ?

Anyway, just call the template `Graph`, and then
```
\brief Test the validity of the graph `g`. [...]
\tparam Graph is a model of `HalfedgeListGraph`
\param g is an object of type `Graph` whose validity is being tested.
```

The `\brief` also needs to describe what is a valid halfedge(list)graph: finite halfedge loops, and reciprocity of associations (see for example the invariants in the BGL doc).

Same for others below.

Reply:
I thought the very interest of having links to a Concept in the doc was to avoid having to re-define it every time we refer to it. Why should I "describe what is a valid halfedge(list)graph" when it is already done in the Concept ?if it is not, maybe it is the Concept page that should be modified.

Reply:
Basically the concept will only tell you what type and functions should be defined for the graph type. I think mentioning the reciprocity with just one example (next <-> prev for example) is sufficient (you get the finite loop for free).